### PR TITLE
Uncheck 'All' filter on start when there is an active filter

### PR DESCRIPTION
**Pull request by yahavi**
- [x] All [tests](https://ci.appveyor.com/project/jfrog-ecosystem/jfrog-idea-plugin) passed. If this feature is not already covered by the tests, I added new tests.
-----

### CODE REVIEW

File: src/main/java/com/jfrog/ide/idea/ui/filters/SelectAllCheckbox.java [hunk 1]
```diff
@@ -35,6 +40,17 @@ void setListeners(@NotNull Map<FilterType, Boolean> selectionMap, @NotNull List<
         });
     }
 
+    /**
+     * Set button checked without trigger the listeners.
```
Comment (eyalbe4) [inline src/main/java/com/jfrog/ide/idea/ui/filters/SelectAllCheckbox.java:44]: trigger --> triggering

File: src/main/java/com/jfrog/ide/idea/ui/filters/FilterMenu.java [hunk 2]
```diff
@@ -37,15 +37,20 @@ public FilterButton getFilterButton() {
...
      * @param selectionMap map between FilterType and boolean that represents whether the filter is checked or not
      */
     void addComponents(@NotNull Map<FilterType, Boolean> selectionMap, boolean putUnknownLast) {
+        setSelectAllCheckbox(selectionMap);
         setListeners(selectionMap);
         addCheckboxes(putUnknownLast);
     }
 
+    private void setSelectAllCheckbox(Map<FilterType, Boolean> selectionMap) {
+        selectAllCheckbox.setChecked(!selectionMap.containsValue(false));
```
Comment (eyalbe4) [inline src/main/java/com/jfrog/ide/idea/ui/filters/FilterMenu.java:51]: Let's add a comment describing the logic behind this.

File: src/main/java/com/jfrog/ide/idea/ui/filters/SelectAllCheckbox.java [hunk 3]
```diff
@@ -14,6 +14,8 @@
  * Created by Yahav Itzhak on 22 Nov 2017.
  */
 class SelectAllCheckbox<FilterType> extends MenuCheckbox {
+    private boolean active = true;
```
Comment (eyalbe4) [inline src/main/java/com/jfrog/ide/idea/ui/filters/SelectAllCheckbox.java:17]: Let's a description for this field.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: src/main/java/com/jfrog/ide/idea/ui/filters/FilterMenu.java (hunk 2 above)
Comment (yahavi) [reply src/main/java/com/jfrog/ide/idea/ui/filters/FilterMenu.java:58]: Added Javadoc

**Pull request merged**